apis/risingwave/v1alpha1/risingwave_types.go

```diff
@@ -456,6 +456,17 @@ type RisingWaveObjectStorageS3 struct {
 	VirtualHostedStyle bool `json:"virtualHostedStyle,omitempty"`
 }
 
+// RisingWaveObjectStorageGCS is the details of GCS bucket storage for compute and compactor components.
+type RisingWaveObjectStorageGCS struct {
```

Comment:
Could the GCS be accessed outside the GCP? If so, then we might need to add fields for credentials here, like the credentials field in the `RisingWaveObjectStorageS3`.

Reply:
Is this something we would like to support? OpenDAL does support using credentials to connect to GCS but the kernel only allows ADC at the moment. This is supported in GKE using workload identity.

Reply:
Done!